builder/comp-builder.nix

```diff
@@ -157,13 +157,14 @@ let
       "--with-ghc=${ghc.targetPrefix}ghc"
       "--with-ghc-pkg=${ghc.targetPrefix}ghc-pkg"
       "--with-hsc2hs=${ghc.targetPrefix}hsc2hs"
-      # CC
+    ] ++ lib.optionals (!stdenv.targetPlatform.isGhcjs)
```

Comment:
We could probably more generically just test for `stdenv.cc == null`, and thus catch all situations where there is no `cc` in the `stdenv`.

Reply:
Yes.